apis/python/tests/test_dataframe.py

```diff
@@ -1899,3 +1899,47 @@ def test_bounds_on_somajoinid_domain(tmp_path):
     )
 
     assert soma.DataFrame.exists(uri)
+
+
+def test_pass_configs(tmp_path, arrow_schema):
+    uri = tmp_path.as_posix()
+
+    with soma.DataFrame.create(uri, schema=arrow_schema()) as sdf:
+        pydict = {}
+        pydict["soma_joinid"] = [0, 1, 2, 3, 4]
+        pydict["foo"] = [10, 20, 30, 40, 50]
+        pydict["bar"] = [4.1, 5.2, 6.3, 7.4, 8.5]
+        pydict["baz"] = ["apple", "ball", "cat", "dog", "egg"]
+        pydict["quux"] = [True, False, False, True, False]
```

Comment:
```suggestion
        pydict["myint"] = [10, 20, 30, 40, 50]
        pydict["myfloat"] = [4.1, 5.2, 6.3, 7.4, 8.5]
        pydict["mystring"] = ["apple", "ball", "cat", "dog", "egg"]
        pydict["mybool"] = [True, False, False, True, False]
```

Feel free to disregard -- this is nitty on my part, & I can do the renames on a separate PR, and you've put a lot of blood, sweat, and tears into this PR already -- I don't want to make things harder for you by making this nitty request